xml_parsing.rb

```diff
@@ -0,0 +1,44 @@
+require 'nokogiri'
+
+class GuiseppesMenu
+
+      attr_accessor :menu, :prices, :calories_array
+
+      def initialize
+        @menu = Nokogiri::XML(File.open('./xml_menu.xml'))
+        @prices = @menu.search('price')
+        @calories = @menu.search('calories')
+        @calories_array = []
+      end
+
+      def removing_pound_sign
+          prices = @prices.text.split ('£')
+          prices.drop(1)
+      end
+
+      def calories_check
+
+        @calories.each do |value|
+        @calories_array << value.text.to_i
+        end
+        @calories_array
+      end
+
+      def calories_above_1000
+        true_values = 0
+        for i in 0..(@calories_array.length-2) do
```

Comment:
In this case, the method is not checking the last one, and it would work only in case of "Full Breakfast" being the last (if not- it wouldn't work properly).